pages/common/ed.md

```diff
@@ -1,36 +1,36 @@
 # ed
 
 > The original Unix text editor.
-> More information: <https://man.archlinux.org/man/ed.1>.
+> More information: <https://manned.org/ed.1>.
 
-- Start ed, editing an empty document (which can be saved as a new file in the current directory):
+- Start an interactive editor session:
```

Comment:
The previous example description seems to be more clear to me.

Reply:
I've changed description to `Start an interactive editor session with empty document`.